src/vscode-dts/vscode.proposed.telemetryLogger.d.ts

```diff
@@ -17,7 +26,7 @@ declare module 'vscode' {
 		 * @param eventName The event name to log
 		 * @param data The data to log
 		 */
-		logUsage(eventName: string, data?: Record<string, string | number | boolean>): void;
+		logUsage(eventName: string, data?: Record<string, any>): void;
```

Comment:
For clarity I'd spec the data as `Record<string, any | TrustedTelemetryValue>`. That provides more guidance how to use it